charts/redpanda/values.schema.json

```diff
@@ -1552,6 +1552,41 @@
           }
         }
       }
+    },
+    "auditLogging": {
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "type": "boolean"
+        },
+        "listener": {
+          "type": "string"
+        },
+        "partitions": {
+          "type": "integer"
+        },
+        "logReplicationFactor": {
+          "type": "integer"
+        },
```

Comment:
This type should be `["integer", "null"]` with `null` being the default.

Reply:
oh thank you, removed.